app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,3 @@
+class PostsController < ApplicationController
+  def index; end
+end
```

Comment:
- You have done well so far, but according to the project requirements, please ensure that you implement the `index` and `show` actions in both controllers. Please fix this 👍 👍